org.eclipse.sisu.plexus.extender/bnd.bnd

```diff
@@ -0,0 +1,2 @@
+Bundle-Name: Sisu-Plexus Extender (Incubation)
```

Comment:
@mcculls I don't understand the need of this empty bundle

Reply:
It's not completely empty - it contains a META-INF/services file which will get picked up by sisu-inject's extender (the OSGi metadata here should have a fragment entry as in the previous manifest.)

For Maven's purposes you can ignore these extender bundles - they exist only to activate certain behaviour in the main bundles. The reason they are separate is to make deployment easier; so users can choose whether to just have the basic behaviour provided by the main bundles, or a more advanced extension mechanism driven by `BundlePlan`s.